universal-login-sdk/src/api/wallet/RequestedMigrating.ts

```diff
@@ -0,0 +1,33 @@
+import {SerializableRequestedMigratingWallet} from '@unilogin/commons';
+import UniLoginSdk from '../sdk';
+import {DeployedWallet} from './DeployedWallet';
+
+export class RequestedMigratingWallet extends DeployedWallet implements SerializableRequestedMigratingWallet {
+  constructor(
+    readonly contractAddress: string,
+    readonly ensName: string,
+    readonly privateKey: string,
+    readonly email: string,
+    public readonly sdk: UniLoginSdk,
+  ) {
+    super(contractAddress, ensName, privateKey, sdk, email);
+  };
```

Comment:
Is it necessary? I think if we don't declare the constructor, we will be able to use DeployedWallet constructor from RequestedMigratingWallet level?